core/openapi/schemas/party.yaml

```diff
@@ -1,103 +1,89 @@
-                  The key name indicates the type of phone number (e.g. home,
-                  work, fax, etc.). The value is the phone number itself.
-                pattern: "^+[1-9]{1}[0-9]{3,14}$"
-          email:
-            type: object
-            description: Email address at which contact can be made.
-            patternProperties:
-              "^.*":
-                type: string
-                description: |-
-                  The key name indicates the type of email address (e.g. home,
-                  work, etc.). The value of the email address itself.
-                format: email
-          address:
-            type: object
-            description: Physical location at which contact can be made.
-            patternProperties:
-              "^.*":
-                type: object
-                description: |-
-                  The key name indicates the type of address (e.g. office,
-                  home, etc.). The value is the address itself.
-                properties:
-                  deliveryPoint:
-                    type: string
-                    description: Address line for the location.
-                  city:
-                    type: string
-                    description: City for the location.
-                  administrativeArea:
-                    type: string
-                    description: State or province of the location.
-                  postalCode:
-                    type: string
-                    description: ZIP or other postal code.
-                  country:
-                    type: string
-                    description: Country of the physical address.
-          url:
-            $ref: common/link.yaml
-            description: On-line information about the responsible party.
-          hoursOfService:
+        description: Telephone numbers at which contact can be made.
+        patternProperties:
+          "^.*":
             type: string
             description: |-
-              Time period (including time zone) when the resposible party
-              can be contacted.
-          contactInstructions:
+              The key name indicates the type of phone number (e.g. home,
+              work, fax, etc.). The value is the phone number itself.
+            pattern: "^+[1-9]{1}[0-9]{3,14}$"
+      email:
+        type: object
+        description: Email address at which contact can be made.
+        patternProperties:
+          "^.*":
             type: string
             description: |-
-              Supplemental instructions on how or when to contact the
-              responsible party.
-      roles:
-        $ref: roles.yaml
+              The key name indicates the type of email address (e.g. home,
+              work, etc.). The value of the email address itself.
+            format: email
+      address:
+        type: object
+        description: Physical location at which contact can be made.
+        patternProperties:
```

Comment:
This line and the one below is equivalent to `additionalProperties`, no?

Reply:
I think the objects are to provide keys, so:

```json
                   "phone": {
                        "office": "+01-819-997-2800"
                    },
                    "email": {
                        "main": "you@example.org",
                        "personal": "foo@example.org"
                    },
```

Reply:
Okay, I don't think it's a good idea thoigh. For example, you can't translate this.

If you use an array with a role (or so) you can change the values depending on the language, but if you change the key, you may break implementations as they may directly access entries through the key.

For example, if you have a multi-language catalog, in english you have:
` "phone": { "office": "+49123456789" },`
In German it gets ugly:
` "phone": { "Büro": "+49123456789" },`

So as long as they keys are not pre-defined and users can put whatever they like, I think this should not be in the keys.

Reply:
Moved to #243